src/app/company/company-add/company-add.component.css

```diff
@@ -0,0 +1,28 @@
+.company-main-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.form-item {
+  width: 30%;
+}
```

Comment:
I think that all **form-item** looks better if they will have an equal box like dimension off **.city-country-box**

Reply:
done